talk/objectorientation/operators.tex

```diff
@@ -86,6 +86,32 @@
   \end{block}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[98]{Friend declarations}
+  \begin{block}{Concept}
+    \begin{itemize}
+      \item Functions/classes can be declared \mintinline{cpp}{friend} within a class scope
+      \item They gain access to all private/protected members
+      \item Useful for operators such as \mintinline{cpp}{a + b}
+      \item Don't abuse friends to go around a wrongly designed interface
+      \item Avoid unexpected modifications of class state in a friend
+    \end{itemize}
+  \end{block}
+  \begin{block}{operator+ as a friend}
```

Comment:
On the other slide, you used a different font:
```suggestion
  \begin{block}{\texttt{operator+} as a friend}
```

Reply:
That's because your suggestion to use `operator<<` was typeset as `operator¡¡`. I unified it now.